### ISSUE TITLE
Mask that always begins with "0.0"

### ISSUE THREAD

**Issue by SPConnelly** [RobinHerbots/Inputmask#2263]
I am trying to make an input mask that always begins with an un-editable 0.0 because the value will never be larger than 0.01. However, a few strange issues are preventing me from doing so. The leading zeros can be edited and the zero digit cannot be typed after the leading zeros.

https://jsbin.com/xukozuniso/edit?html,js,output

Any help to figure out what is going wrong is much appreciated.

**Comment by RobinHerbots** [comment]
@SPConnelly , 

try this mask
 ```
new Inputmask({
  mask: '0.09{0,3}',
}).mask(document.querySelector("input"));
```

**Comment by SPConnelly** [comment]
@RobinHerbots 

That is a better way to give optional precision but the "0.0" part is still editable. I thought anything other than '9' was to remain the same in the mask. Also, weird behavior occurs when I try to type a zero after "0.0" in any of the optional decimal places.

I have improved my snippet slightly: https://jsbin.com/gimawahiza/edit?html,js,output

In this example everything seems to work fine, "0.0" is static, even if I backspace over it and begin typing, what I type appears after it. I believe that double escaping the zeroes is exactly what I needed here.

However, I run into another problem if I try to type a zero directly after the static "0.0". The zero appears to be typed normally but the caret has been moved to the decimal point. If another digit is typed at this point (say 1 for example) it will then be placed at the end of the optional decimal places and the remaining places are filled with zero.

A similar issue to the previous occurs if I type 1 followed by 0, I would expect to see "0.010", but the 1 is placed at the end of the optional decimal places and the rest are filled with zero resulting in "0.0001".
